src/BlueSCSI_disk.cpp

```diff
@@ -356,10 +356,11 @@ class ImageBackingStore
         }
         else
         {
-            if(type == S2S_CFG_OPTICAL)
+            m_isreadonly_attr = !!(FS_ATTRIB_READ_ONLY & SD.attrib(filename));
```

Comment:
Is the "not not" here doing anything useful?

Reply:
Converting an int to a c++ bool it seems https://stackoverflow.com/questions/6764168/what-does-double-not-used-on-a-non-boolean-variable-do